frontend/src/components/marketDetails/MarketDetailsLayout.jsx

```diff
@@ -1,75 +1,105 @@
 import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
-import MarketChart from '../charts/MarketChart';
 import { formatDateTimeForGrid } from '../utils/dateTimeTools/FormDateTimeTools';
-import DescriptionButton from '../buttons/marketDetails/DescriptionButton';
+import MarketChart from '../charts/MarketChart';
+import ActivityTabs from '../../components/tabs/ActivityTabs';
+
+function MarketDetailsTable({
+  market,
+  creator,
+  numUsers,
+  totalVolume,
+  currentProbability,
+  probabilityChanges,
+  marketId,
+}) {
+  const [showFullDescription, setShowFullDescription] = useState(false);
+
+  return (
+    <div className='bg-gray-900 text-gray-300 p-4 rounded-lg shadow-lg w-full overflow-hidden'>
+      <div className='mb-4'>
+        <h1 className='text-xl font-semibold text-white mb-2 break-words'>
+          {market.questionTitle}
```

Comment:
I'm showing for large market titles, (e.g. 160 characters) there is an overlap which breaks the page. Can we do a text wrap here too?

![image](https://github.com/user-attachments/assets/4d61ff08-0d07-4cc1-ba79-123c71b4a647)

Reply:
On it

Reply:
Fixed here https://github.com/openpredictionmarkets/socialpredict/pull/242/commits/11046dae72a8591a19d29a6d71ddd5b1b5a7c189

Reply:
OK, I'm going to resolve this because my test case is 160 letter word rather than 160 total characters which is not a common situation.